tests/test_api.py

```diff
@@ -227,7 +229,7 @@ async def test_list_events(
     )
     assert url_request() == [
         f"/calendars/some-calendar-id/events?{EVENT_LIST_PARAMS}"
-        "&timeMin=2022-04-30T01:31:02%2B00:00"
+        "&timeMin=2022-04-30T01:31:02Z"
```

Comment:
The default repr format from pydantic changed now that `v2` is using `pydantic_core`. As the new one also follows RFC3339, it should be compatible with the API.
https://developers.google.com/workspace/calendar/api/v3/reference/events/list

Reply:
I tested this with the API and it works.